.gitignore

```diff
@@ -0,0 +1,7 @@
+# Logs and databases #
+######################
+ui/logs/
+ui/__pycache__/
+*.sql
+*.sqlite
+*.db
```

Comment:
I also get updates from 
`.../git/portfolio/ui/app.db`

also a few others. For example: `.../git/portfolio/ui/client/__pycache__/errors.cpython-38.pyc`. 

Should I be paying attention to these?